pkg/config/config.go

```diff
@@ -522,3 +525,47 @@ func GetSrcType(ctStr string) (SrcType, error) {
 		return NONE, fmt.Errorf("Unsupported component source type: %s", ctStr)
 	}
 }
+
+// GetOSSourcePath corrects the current sourcePath depending on local or binary configuration,
+// if Git has been passed, we simply return the source location from LocalConfig
+// this will get the correct source path whether on Windows, macOS or Linux.
+func GetOSSourcePath(localConfig *LocalConfigInfo) (path string, err error) {
```

Comment:
Can we align it with the rest of Get functions and use LocalConfig as a receiver?
```go
func (lc *LocalConfig) GetOSSourcePath()
```

Reply:
Done.